features/layout/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -153,9 +153,42 @@ const CollapseMenuItem = styled(MenuItemButton)`
 `;
 
 export function SidebarNavigation() {
+  const bodyEl =
+    typeof window !== "undefined" ? document.querySelector("body") : null;
+  const breakpointDesktop = 1024;
   const router = useRouter();
   const { isSidebarCollapsed, toggleSidebar } = useContext(NavigationContext);
+  const [mounted, setMounted] = useState(false);
   const [isMobileMenuOpen, setMobileMenuOpen] = useState(false);
+  const [isMobileScreen, setMobileScreen] = useState(
+    bodyEl != undefined && bodyEl.clientWidth >= breakpointDesktop
+      ? false
+      : true
+  );
+
+  const setScreenSize = () => {
+    if (bodyEl != undefined) {
+      // Desktop screen
+      if (bodyEl.clientWidth >= breakpointDesktop) {
+        setMobileScreen(false);
+      }
+      // Mobile screen
+      else {
+        setMobileScreen(true);
+      }
+    }
+  };
+
+  useEffect(() => {
+    setMounted(true);
+    window.addEventListener("resize", setScreenSize);
+
+    return () => {
+      window.removeEventListener("resize", setScreenSize);
+    };
+  }, []);
```

Comment:
This comment by Github is correct. `setScreenSize` is missing from the dependencies here which could lead to bugs in case the `bodyEl` is not set during the first render (I guess that wouldn't happen but still)